src/Model/GroundWaterFlow/gwf3sto8.f90

```diff
@@ -312,22 +350,35 @@ subroutine sto_fc(this, kiter, hold, hnew, njasln, amat, idxglo, rhs)
       !
       ! -- set saturation used for ss
       ss_sat1 = snnew
-      ssh0 = hold(n)
-      ssh1 = DZERO
```

Comment:
The volume of water released or added to specific storage needs to be based on the top of the cells. I am not sure this is correct.